### PR TITLE
update devnet

### PR DESCRIPTION
**Pull request by opfocus**
**Description**
This update is in response to changes on the develop branch.

**Tests**
Tested:
op-challenger logs:
```
t=2024-08-16T18:27:43+0000 lvl=info msg="Publishing transaction" service=challenger tx=0xf8f8eb89a35e3be0ce888a438769c286e58e9c4bdf61bf03fc79098dcafaed89 nonce=12 gasTipCap=1000000000 gasFeeCap=3000000000 gasLimit=128937 tx=0xf8f8eb89a35e3be0ce888a438769c286e58e9c4bdf61bf03fc79098dcafaed89
t=2024-08-16T18:27:43+0000 lvl=info msg="Transaction successfully published" service=challenger tx=0xf8f8eb89a35e3be0ce888a438769c286e58e9c4bdf61bf03fc79098dcafaed89 nonce=12 gasTipCap=1000000000 gasFeeCap=3000000000 gasLimit=128937 tx=0xf8f8eb89a35e3be0ce888a438769c286e58e9c4bdf61bf03fc79098dcafaed89
t=2024-08-16T18:27:55+0000 lvl=info msg="Transaction confirmed" service=challenger tx=0xf8f8eb89a35e3be0ce888a438769c286e58e9c4bdf61bf03fc79098dcafaed89 block=0xfb60dcceb1a6eaa896dbba9c6d85046d9d803b662a12de69ca1be9ebafedfb73:39 effectiveGasPrice=1005694214
t=2024-08-16T18:27:55+0000 lvl=info msg="Resolving game" game=0x490319228F94cd58Ee8071bb3e37090aF1764f36
t=2024-08-16T18:27:55+0000 lvl=info msg="Publishing transaction" service=challenger tx=0x79551c9249248721be7c2760fa23715f5cc1780175b6af1aabbd90649f4a1665 nonce=13 gasTipCap=1000000000 gasFeeCap=3000000000 gasLimit=74590 tx=0x79551c9249248721be7c2760fa23715f5cc1780175b6af1aabbd90649f4a1665
t=2024-08-16T18:27:55+0000 lvl=info msg="Transaction successfully published" service=challenger tx=0x79551c9249248721be7c2760fa23715f5cc1780175b6af1aabbd90649f4a1665 nonce=13 gasTipCap=1000000000 gasFeeCap=3000000000 gasLimit=74590 tx=0x79551c9249248721be7c2760fa23715f5cc1780175b6af1aabbd90649f4a1665
```

**Additional context**

Add any other context about the problem you're solving.

**Metadata**

- Fixes #[Link to Issue]

### CODE REVIEW

File: pages/chain/testing/dev-node.mdx [hunk 1]
```diff
@@ -40,10 +40,13 @@ Note that the command line directions were verified under Ubuntu 22.04 LTS.
 Other OSes or versions may use different tools.
 
 <Steps>
-  ### Install the command line utilities `make` and `jq`
+  ### Install the command line utilities `make`, `jq`, and `just`. For Docker installation, please refer to the instructions [here](https://docs.docker.com/engine/install/ubuntu/).
 
   ```sh
   sudo apt install -y make jq
+  wget https://github.com/casey/just/releases/download/1.34.0/just-1.34.0-x86_64-unknown-linux-musl.tar.gz
```
Comment (anacrolix) [inline pages/chain/testing/dev-node.mdx:47]: I don't think you should specific an exact version or installation process in the docs. Just link to the best place with instructions and mention where to find out what version is required?

File: pages/chain/testing/dev-node.mdx [hunk 2]
```diff
@@ -54,6 +57,7 @@ Other OSes or versions may use different tools.
   tar xvzf go1.21.5.linux-amd64.tar.gz
   sudo cp go/bin/go /usr/bin/go
   sudo mv go /usr/lib
+  echo export PATH=$PATH:/root/go/bin >> ~/.bashrc
```
Comment (anacrolix) [inline pages/chain/testing/dev-node.mdx:60]: Same as above, I think it's better to defer how to install specific packages to their native documentation, unless we need something unusual.

**Comment by opfocus** [comment]
This might be a matter of style preference. I made these changes simply to update the document, which is currently outdated. I updated it in a way that continues the original style. Since I'm not a native English speaker, I'm not skilled at making stylistic changes to the text

File: pages/chain/testing/dev-node.mdx (hunk 2 above)
Comment (opfocus) [reply pages/chain/testing/dev-node.mdx:60]: I hope someone can help edit it. If I'm not mistaken, the installation link is https://go.dev/doc/install

File: pages/chain/testing/dev-node.mdx (hunk 1 above)
Comment (opfocus) [reply pages/chain/testing/dev-node.mdx:47]: I have enabled the "Allow edits and access to secrets by maintainers" option. I hope someone can help edit it. If I'm not mistaken, the installation link is https://github.com/casey/just.

**Comment by anacrolix** [comment]
Thanks for the contribution, someone will check it soon.

File: pages/chain/testing/dev-node.mdx (hunk 1 above)
Comment (sbvegan) [reply pages/chain/testing/dev-node.mdx:47]: Hey @opfocus, I'm going to unresolve these conversations for visibility by another docs reviewer

**Comment by opfocus** [comment]
Additional:Installation methods are also mentioned in other locations: https://github.com/ethereum-optimism/superchain-ops .

**Review by cpengilly — APPROVED** [review]

File: pages/chain/testing/dev-node.mdx (hunk 1 above)
Comment (krofax) [reply pages/chain/testing/dev-node.mdx]: I've reviewed the docs.

File: pages/chain/testing/dev-node.mdx [hunk 3]
```diff
@@ -35,28 +34,51 @@ Someone nice will help you out!
...
+    *   [Install just](https://github.com/casey/just):
+
+    </Tabs.Tab>
+
+    <Tabs.Tab>
+      ### Mac Installation
+
+    *   Install Homebrew (if not already installed):
+
+      ```sh
```
Comment (bradleycamacho) [inline pages/chain/testing/dev-node.mdx:61]: I'd link out to homebrew instead of having a command that might go out of date at some point: https://brew.sh/

File: pages/chain/testing/dev-node.mdx [hunk 4]
```diff
@@ -35,28 +34,51 @@ Someone nice will help you out!
...
-  sudo cp go/bin/go /usr/bin/go
-  sudo mv go /usr/lib
-  echo export GOROOT=/usr/lib/go >> ~/.bashrc
-  . ~/.bashrc
-  ```
+      ```sh
+      sudo apt install -y make jq
+      ```
+
+    *   [Install just](https://github.com/casey/just):
```
Comment (bradleycamacho) [inline pages/chain/testing/dev-node.mdx:52]: I'd place this bullet above the `make` and `jq` bullet. The command breaks up the ordering a bit and it's a bit hard to parse

File: pages/chain/testing/dev-node.mdx [hunk 5]
```diff
@@ -35,28 +34,51 @@ Someone nice will help you out!
...
-  ```sh
-  sudo apt update
-  wget https://go.dev/dl/go1.21.5.linux-amd64.tar.gz
-  tar xvzf go1.21.5.linux-amd64.tar.gz
-  sudo cp go/bin/go /usr/bin/go
-  sudo mv go /usr/lib
-  echo export GOROOT=/usr/lib/go >> ~/.bashrc
-  . ~/.bashrc
-  ```
+      ```sh
```
Comment (bradleycamacho) [inline pages/chain/testing/dev-node.mdx:48]: You might have to toy around with this, but can we make the code box live on the same level of indentation as the bullet? You might just need to delete line 47 to make it work, but I'm not entire sure

File: pages/chain/testing/dev-node.mdx [hunk 6]
```diff
@@ -65,30 +87,24 @@ Other OSes or versions may use different tools.
...
-  ```
-  
   ### Install Foundry using monorepo tooling
 
-  The following command will install Foundry if you don't already have it on your system.
+  The following command will install Foundry if it is not already installed on your system. If an error occurs during execution, please run `source /root/.bashrc` and try again.
 
   ```sh
-  pnpm install:foundry
+  just install-foundry
```
Comment (bradleycamacho) [inline pages/chain/testing/dev-node.mdx:95]: Did we verify we should be using `just` instead of `pnpm`?

**Review by bradleycamacho — APPROVED** [review]

File: pages/chain/testing/dev-node.mdx (hunk 6 above)
Comment (krofax) [reply pages/chain/testing/dev-node.mdx:95]: Yes we have

File: pages/chain/testing/dev-node.mdx (hunk 5 above)
Comment (bradleycamacho) [reply pages/chain/testing/dev-node.mdx:48]: To clarify- the code block currently "breaks" the list and it seems seperate from the bullet it is connected to. If we indent it correctly, it'll visually be part of the bullet above. 

Here's an example from another site of a block formatted correctly and one that isn't

Screenshot 2024-08-22 at 9 37 21 AM
